src/Microsoft.AspNet.SignalR.Core/Messaging/ScaleoutSubscription.cs

```diff
@@ -100,9 +100,6 @@ protected override void PerformWork(IList<ArraySegment<Message>> items, out int
                 ScaleoutMapping mapping = enumerator.Current.Item1;
                 int streamIndex = enumerator.Current.Item2;
 
-                _trace.TraceVerbose("Extracting message for scaleout mapping: {0}, stream index {1}, [{2}]",
```

Comment:
Extra noisy traces I had added in an earlier commit for debugging that need to be removed.

Reply:
Are all the other traces you added quiet enough to keep?

Reply:
All the other traces were removed in earlier commits ;)

Reply:
But I'll do a quick pass to sanity check that.